distributed/distributed.yaml

```diff
@@ -176,7 +176,7 @@ distributed:
       threads: 0    # Threads to use. 0 for single-threaded, -1 to infer from cpu count.
 
     timeouts:
-      connect: 10s          # time before connecting fails
+      connect: 20s          # time before connecting fails
```

Comment:
There is a PR open for ages now to increase this. https://github.com/dask/distributed/pull/4228

Reply:
Bumped to 30s